src/components/Page/ProgressModalPage.stories.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 AccelByte Inc. All Rights Reserved.
+ * This is licensed software from AccelByte Inc, for limitations
+ * and restrictions contact your company contract manager.
+ */
+
+import React from "react";
+import { Meta, Story } from "@storybook/react/types-6-0";
+import { ProgressModalPage, ProgressModalPageProps } from "./ProgressModalPage";
+
+export default {
+  title: "Components/ProgressModalPage",
+  component: ProgressModalPage,
+} as Meta;
+
+const Template: Story<ProgressModalPageProps> = (args) => <ProgressModalPage {...args} />;
+
+export const Example = Template.bind({});
+Example.args = {
+  title: "Progress Indicator Title",
+  step: 3,
+  stepLabels: ["Step 1", "Step 2", "Step 3"],
+  children: <p>Some children component</p>,
```

Comment:
I see the `onClose` props as mandatory in `ModalPage` I think we should put the props here too. You can print close modal message, or put history.back there.

Reply:
we don't have history in ui-lib. it's kinda AP specific if you look at src/utils/history, so i'll just add console.log here

Reply:
i update again to set onClose as optional though since in modalpage AP, it's also optional

Reply:
I believe in the AP is optional because by default they handle the close modal with `history.back`, meanwhile in here we don't have the history function.

Reply:
if we make it required, we will get unnecessary error warning in IDE when we implement this component in ModalPage file in AP

Reply:
anyway, i make it optional in ModalPage, but required in ProgressModalPage